examples/update-fungible-token-metadata-with-admin-key.js

```diff
@@ -0,0 +1,105 @@
+import {
+    TokenCreateTransaction,
+    TokenInfoQuery,
+    TokenType,
+    PrivateKey,
+    Client,
+    AccountId,
+    TokenUpdateTransaction,
+} from "@hashgraph/sdk";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * @notice E2E-HIP-646
+ * @url https://hips.hedera.com/hip/hip-646
+ * @description Update fungible token metadata with admin key
+ */
+async function main() {
+    if (
+        !process.env.OPERATOR_KEY ||
+        !process.env.OPERATOR_ID ||
+        !process.env.HEDERA_NETWORK
+    ) {
+        throw new Error("Please set required keys in .env file.");
+    }
+
+    const operatorId = AccountId.fromString(process.env.OPERATOR_ID);
+    const operatorKey = PrivateKey.fromStringDer(process.env.OPERATOR_KEY);
+    const network = process.env.HEDERA_NETWORK;
+    const client = Client.forName(network).setOperator(operatorId, operatorKey);
+
+    // Generate a supply key
+    const adminKey = PrivateKey.fromStringDer(process.env.OPERATOR_KEY);
+    // Initial metadata
```

Comment:
we should generate a new admin key that is different that the operator key. The operator key in theory is the key of the account that is paying for the transaction fees and not associated with the admin key type functionality.